docs/reference/generated/slider-thumb.json

```diff
@@ -6,29 +6,106 @@
       "type": "string",
       "description": "The label for the input element."
     },
+    "aria-labelledby": {
+      "type": "string"
+    },
+    "name": {
+      "type": "string"
+    },
+    "active": {
+      "type": "number",
+      "description": "The index of the active thumb."
+    },
     "aria-valuetext": {
       "type": "string",
       "description": "A string value that provides a user-friendly name for the current value of the slider."
     },
     "getAriaLabel": {
-      "type": "function(index: number) => string",
+      "type": "null | ((index: number) => string)",
       "description": "Accepts a function which returns a string value that provides a user-friendly name for the input associated with the thumb"
     },
     "getAriaValueText": {
-      "type": "function(formattedValue: string, value: number, index: number) => string",
+      "type": "null | ((formattedValue: string, value: number, index: number) => string)",
       "description": "Accepts a function which returns a string value that provides a user-friendly name for the current value of the slider.\nThis is important for screen reader users."
     },
+    "handleInputChange": {
```

Comment:
Hmm, why were these missing before?

Reply:
Good question. Could be a bug in the previous implementation?